Gemfile.lock

```diff
@@ -353,4 +353,4 @@ RUBY VERSION
    ruby 2.3.0p0
 
 BUNDLED WITH
-   1.12.4
+   1.12.5
```

Comment:
relative with the above comment, you should not update the `Gemfile.lock` since you are not adding or updating any gem